src/plugins/analysis/linter/code/source_code_analysis.py

```diff
@@ -119,8 +129,7 @@ def analyze(self, file_handle: io.FileIO, virtual_file_path: str, analyses: dict
         )
 
 
-def run_linguist(file_path: str) -> dict:
-    file_name = Path(file_path).name
+def run_linguist(file_path: str, file_name: str) -> dict:
```

Comment:
Can we rename the second argument to virtual_name or virtual_file_name or sth?
Right now from looking at the signature I'd think both arguments are the same thing.

Reply:
changed